UnityProject/Assets/Scripts/Messages/Server/TabUpdateMessage.cs

```diff
@@ -120,7 +120,7 @@ public static void SendToPeepers(GameObject provider, NetTabType type, TabAction
 					var instance = NetworkTabManager.Instance.Get(provider, type);
 					if (instance == null)
 					{
-						Logger.LogError($"Couldn't find NetTab to send! " +
+						Logger.LogError($"Couldn't find NetTab to send for {provider.ExpensiveName()} " +
 								$"Does the tab prefab match the type '{type}'? Expecting prefab name Tab'{type}' in a specific folder.");
```

Comment:
```suggestion
								$"Does the tab prefab match the type '{type}'? Expecting prefab name 'Tab{type}' in a specific folder.");
```
Just fixing my mistake...

Edit: dang it, didn't want to comment this until just before I merged, but the other comment submitted this one as well ; )